app/controllers/sign_up/passwords_controller.rb

```diff
@@ -58,7 +65,8 @@ def process_successful_password_creation
     end
 
     def password_form
-      @password_form ||= PasswordForm.new(@user)
+      validate_confirmation = true
+      @password_form ||= PasswordForm.new(@user, validate_confirmation)
```

Comment:
I like that the use of a named variable helps avoid ambiguity from inlining the `true` argument, but it seems like a keyword argument is well-suited for that?

```suggestion
      @password_form ||= PasswordForm.new(@user, validate_confirmation: true)
```